src/styles.css

```diff
@@ -1 +1,2 @@
 /* You can add global styles to this file, and also import other style files */
+@import "~bootstrap/dist/css/bootstrap.css";
```

Comment:
This must be in angular.json. Find out an alternative for this. Do not import css from local files here

Reply:
But what is wrong in importing bootstrap like this, as style.css is meant for this purpose? We can import style files as the comment suggests

Reply:
No doubt you can import file like this.
In your case, style.css will be downloaded first and then "~bootstrap/dist/css/bootstrap.css"; will be downloaded.
It is good practice to add files in angular.json so that we can take advantage while downloading file in browser.

Reply:
Okay, I have moved it to angular.json